zcash_transparent/src/zip48.rs

```diff
@@ -420,6 +482,20 @@ mod tests {
                 assert_eq!(&key.key_info_expression(&params), expected);
             }
 
+            for (i, seed) in (0..tv.key_information_vector.len()).zip(&seeds) {
+                if i < tv.key_information_vector.len() {
+                    assert!(matches!(
+                        fvk.derive_matching_account_priv_key(seed),
+                        Ok(Some(_)),
+                    ));
+                } else {
```

Comment:
blocking: How do we ever get to this `else` clause? We zipped `0..tv.key_information_vector.len()` with `&seeds` but `zip` halts at the end of the shorter collection, so AFAICT we can never get an index that would trigger this `else`.